src/Libraries/CoreNodeModels/Logic/If.cs

```diff
@@ -58,6 +63,8 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
             }
             else
             {
+                UseLevelAndReplicationGuide(inputAstNodes);
```

Comment:
@Amoursol @Racel  FYI - I wonder why this was left off in the first place, any memory?

Reply:
@aparajit-pratap I think we need to be careful adding this in this task - what if it changes the behavior of the IF node in existing graphs - in the default constructor do we need to set the lacing? Won't that come from the deserialized node?

Reply:
The default will be auto mode, which is the same if lacing is absent altogether so I don't think it will change the behavior of existing nodes. You're right about the deserialized node. Only if the lacing info is absent it should be sent to auto otherwise it should be read from the file. Will make the change.

Reply:
Hmm, it looks like this change does not affect existing nodes at all (neither xml nor json files). The `If` node if deserialized from an old file does not have the lacing option at all just like before. In a nutshell this change will affect only new nodes made from this point on.